### ISSUE TITLE
How I train d-vector model for using on diarization with my own data?

### ISSUE THREAD

**Issue by mesut92** [HarryVolek/PyTorch_Speaker_Verification#67]
Hi Harry;
I want to use d-vector for diarization with 8kHz data. I have 9000 speakers. However my loss saturate around 5 (at 250 epoch)(Should I train with more epochs?). I use NIST data (it's around 400GB). I can not get enough performance in diarization. Do you have any suggestions?
Best regards;
Thanks
Mesut

**Comment by Gaurav470** [comment]
Hi @mesut92 ,
I trained my speaker verification model on 5K speakers and used this model to get d-vector embeddings and trained the UIS-RNN model on these embeddings. Then created embeddings of wav files i needed to get the prediction of. But i am only get a single speaker for all the wav files when i am sure it has multiple speakers.
Thanks in advance.

**Comment by asr-lord** [comment]
> Hi @mesut92 ,
> I trained my speaker verification model on 5K speakers and used this model to get d-vector embeddings and trained the UIS-RNN model on these embeddings. Then created embeddings of wav files i needed to get the prediction of. But i am only get a single speaker for all the wav files when i am sure it has multiple speakers.
> Thanks in advance.

I've the same problem, how did you solve it @Gaurav470 ?
